### PR TITLE
Fixes #8150: Update does not work in package_install_version_cmp_update

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8150

### CODE REVIEW

**Comment by amousset** [comment]
Do not merge for now, still testing.

File: tree/30_generic_methods/package_install_version_cmp_update.cf [hunk 1]
```diff
@@ -42,8 +42,8 @@ bundle agent package_install_version_cmp_update(package_name, version_comparator
   defaults:
       "action" string => "immediate", if_match_regex => "add";
       "action" string => "warn", if_match_regex => "verify|";
-      "update_policy" string => "addupdate", if_match_regex => "true";
-      "update_policy" string => "add",       if_match_regex => "false";
+      "update_policy" slist => {"add", "update"}, if_match_regex => "true";
```
Comment (ncharles) [inline tree/30_generic_methods/package_install_version_cmp_update.cf]: this is a clever idea :)
I hope it works

**Comment by jooooooon** [comment]
Ready for TR, please review and merge

**Comment by peckpeck** [comment]
Seems OK for me

**Comment by jooooooon** [comment]
So let's merge it then?

**Pull request merged**